### PR TITLE
home page updates opening up general sponsorship

### PR DESCRIPTION
**Pull request by jodygarnett**
Previously we just highlighted *OGC Implementation* as benefiting form sponsorship, wanted to fix that.

OGC Implementation:

* Have a text asking for both financial support and developers

Part of a vibrant open source community:

* Restore foss4g text and added FOSS4G 2022 Firenzel link

Special Thanks:

* Removed Packt Publishing books section as we have not received any funds
* Replaced with general sponsorship section with some words about sustainability and underfunded activities
* Made the section on in-kind development tools more clear and added *Jira Software* so it is not just YourKIT

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,34 +118,44 @@ <h4>OGC Implementation</h4>
...
+      <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4XZ76BZMEWZ24&source=url">
+        <img src="img/btn_donateCC_LG.gif" width="147" height="47"/>
+      </a>
+    </div>
+    <div class="col-md-6" id="in-kind">
+      <p class="text-left">
+         GeoServer benefits the following development tools via their in-kind support of  open source projects.
+      </p>
+      <a href="https://www.yourkit.com/java/profiler/index.jsp"><img src="img/yklogo.png" width="150" height="36"/></a>
+      <a href="https://www.yourkit.com/java/profiler/index.jsp"><img src="img/jira.png" width="150" height="20"/></a>
```
Comment (bradh) [inline index.html:158]: Do we want to link to Atlassian here?

Comment (jodygarnett) [reply index.html:158]: Oh yeah we do, sorry was focused on the visual

Comment (jodygarnett) [inline index.html:158]:
```suggestion
      <a href="https://www.atlassian.com/software/jira"><img src="img/jira.png" width="150" height="20"/></a>
```

File: index.html [hunk 2]
```diff
@@ -118,34 +118,44 @@ <h4>OGC Implementation</h4>
...
+        GeoServer is working with the OSGeo Foundation to be certified as compliant and we need your help. Both <a href="https://github.com/geoserver/geoserver/wiki/Sponsor">financial support</a> and <a href="https://github.com/geoserver/geoserver/wiki/Restore-CITE-Certification">developers</a> are needed to complete this activity.
     </div>
     <div class="col-md-6">
       <h4>Part of a Vibrant Open Source Community</h4>
       <p class="text-left">
         GeoServer is developed, tested, and supported as community-driven project by a diverse group of individuals and organizations around the world. We are proud to be a recognized <a href="https://osgeo.org/projects/geoserver/">Open Source Geospatial Foundation</a> project.
       </p>
-      <a href="https://osgeo.org/projects/geoserver/"> <img src="img/OSGeo_project.png"/ width="150"></a>
+      <p class="text-left">
+        GeoServer participates in the annual <a href="https://foss4g.org/">Free and Open Source Software for Geospatial</a> conference, Join us for <a href="https://osgeo.org/projects/geoserver/">FOSS4G 2022 - Firenze</a>!
```
Comment (bradh) [inline index.html:129]:
```suggestion
        GeoServer participates in the annual <a href="https://foss4g.org/">Free and Open Source Software for Geospatial</a> conference, join us for <a href="https://2022.foss4g.org/">FOSS4G 2022 - Firenze</a>!
```

File: index.html [hunk 3]
```diff
@@ -118,34 +118,44 @@ <h4>OGC Implementation</h4>
...
-       <a href="https://www.yourkit.com/java/profiler/index.jsp"><img src="img/yklogo.png"/ ></img></a>
+          Thanks for your <a href="https://github.com/geoserver/geoserver/wiki/Sponsor">sponsorship</a> and <a href="https://github.com/sponsors/OSGeo">github</a> / <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4XZ76BZMEWZ24&source=url">paypal</a> donations. Financial support contributes to project sustainability by supporting underfunded <a href="https://github.com/geoserver/geoserver/wiki/Maintenance">maintenance</a>, <a href="https://github.com/geoserver/geoserver/blob/main/SECURITY.md">security</a>, and <a href="https://github.com/geoserver/geoserver/wiki/Restore-CITE-Certification">interoperability</a> activities.
+      </p>
+      <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4XZ76BZMEWZ24&source=url">
+        <img src="img/btn_donateCC_LG.gif" width="147" height="47"/>
+      </a>
+    </div>
+    <div class="col-md-6" id="in-kind">
+      <p class="text-left">
+         GeoServer benefits the following development tools via their in-kind support of  open source projects.
```
Comment (bradh) [inline index.html:155]: The sentence doesn't really match up - we don't benefit the tools, we benefit from them, but the tools aren't providing support - the companies are.
Maybe
```suggestion
         GeoServer is one of many open-source projects that receives in-kind support from:
```

**Pull request merged**